modules/upgrade/pages/client-y-z.adoc

```diff
@@ -0,0 +1,106 @@
+When you have many {sles} clients to manage, you can automate in-place upgrades using the content lifecycle manager.
+
+
+
+== Prepare to Upgrade
+
+Before you can upgrade your clients, you need to make these preparations:
+
+* Create a content lifecycle project
+* Create an activation key
+* Create an autoinstallation distribution
+* Create an autoinstallation profile
+
+.Procedure: Creating a Content Lifecycle Project
+. Create a content lifecycle project for your distribution.
+For more information, see xref:administration:content-lifecycle.adoc[].
+. Ensure you a choose a short but descriptive name for your project.
+. Include all source channel modules that you require for your distribution.
+. Add filters as required, and set up at least one environment.
+
+
+
+.Procedure: Creating an Activation Key
+. Create an activation key for your distribution.
+For more information, see xref:client-configuration:clients-and-activation-keys.adoc[].
+. Ensure your activation key includes all filtered project channels.
+
+
+
+.Procedure: Creating an Autoinstallation Distribution
+. Create an autoinstallation distribution for every base channel you want to migrate.
+For more information, see xref:client-configuration:client-automating-preparation.adoc[].
+. Give your distribution a label that references the name of the content lifecycle project.
+. In the ``Installer Generation`` field, select the SLES version you are using.
+
+
+
+.Procedure: Creating an Autoinstallation Profile
+. Create an autoinstallation profile for every target distribution and service pack you want to upgrade to.
+For more information, see xref:client-configuration:client-automating-preparation.adoc[].
+. You will require a different profile for Salt and traditional clients.
+. You can use variables in the profile to distinguish between the different lifecycle environments.
+
+
+For example autoinstallation profiles, see xref:upgrade:upgrade-script-example.adoc[].
+
+Use these variables in your autointallation profiles for automating in-place upgrades:
+
+
+
+.Example: Variables for use in Autoinstallation Profiles
+----
+registration_key=1-15sp1-demo-test
+org=1
+channel_prefix=15sp1-demo-test
+distro_label=15sp1-demo-test
+----
+
+
+
+.Example: Entry for use in Autoinstallation Profiles
+----
+ <listentry>
+     <ask_on_error config:type="boolean">true</ask_on_error>
+     <media_url>https://$redhat_management_server/ks/dist/child/$channel_prefix-sle-module-web-scripting15-sp1-pool-x86_64/$distro_label</media_url>
+     <name>$channel_prefix SLE-Module-Web-Scripting15-SP1 Pool for x86_64 </name>
+     <product>Web Scripting Module 15 SP1 x86_64 Pool</product>
+</listentry>
+----
+
+
+
+== Upgrade
+
+
+When you have prepared the server for the upgrade, you can provision the clients.
+
+
+
+.Procedure: Provisioning the Clients
+. In the {productname} {webui}, navigate to menu:Systems[System List], and select the clients you want to provision to add them to the system set manager.
```

Comment:
This "to ... to ... to" is not that easy to understand.  Here is an attempt to split the sentence:
```suggestion
. In the {productname} {webui}, navigate to menu:Systems[System List], and select the clients you want to provision.
Selecting them you add them to the system set manager.
```

Reply:
That doesn't quite work either. What about:

> Selecting the clients adds them to the system set manager.


??

Reply:
I'm not sure about the right wording, but I think it works as follows:

1/ You select the clients (you tick them)
2/ Then all selected clients are ready to be managed by system set manager routines

Maybe better just ignore my suggestion.  I think the initial is technically correct and we can keep it. ;)

Reply:
I think any technical differences are pretty minor here. I merged it all into one step because we describe it in detail in the SSM chapter, and it's the kind of task you are likely to do over and over again.